README.md

```diff
@@ -73,14 +52,42 @@ npm run start
 
 Lunie is automatically connecting to the backend at `http://localhost:4000`. If not set the according environment variable `VUE_APP_GRAPHQL_URL`.
 
-The testnet will contain a rich account that you can use to see balances and make transactions. You need to import that account into Lunie:
+The testnet will contain an account with plenty of funds inside that you can use to see balances and make transactions. You need to import that account into Lunie:
+
 - Sign In
 - Use an existing account
 - Recover with backup code
 
-Now enter the backup code and create the account. Backup code: `release endorse scale across absurd trouble climb unaware actor elite fantasy chair license word rare length business kiss smoke tackle report february bid ginger`
+Now enter the backup code and create the account. Backup code: 
+
+`release endorse scale across absurd trouble climb unaware actor elite fantasy chair license word rare length business kiss smoke tackle report february bid ginger`
+
+You should now have a bunch of funds to stake to play with.
+
+
+[comment]: <> (moved Ledger section further down the page to keep the instructions of installing dependencies in the same general area)
```

Comment:
?

Reply:
Commented how I wanted to move it further down the page because it wasn't immediately relevant to installing.